### DISCUSSION TITLE
Welcome to SuperPrompt Discussions!

### DISCUSSION THREAD

**Discussion by NeoVertex1** [NeoVertex1/SuperPrompt#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by helderka** [comment]
Hi Neo, the truth that I would like to share that the prompt has fascinated me, conversations with a tendency to depth are a sea of opportunities for new thoughts. I am honestly impressed with each of the conversations. To give an example, take a picture of a flower and reflect on it ...  I would like to know what the full story is behind superprompt; and honestly thanks for your contribution, it was a success to find your repository .... that I can't remember how I found it.

**Reply by NeoVertex1** [reply]
thank you so much for the kind words, im really working still to try to help others grasp better, I want to write probablya small book about everythin I know about prompts some kind of lenghty paper or something, im also gonna make a list of readings for people
